dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05390RemoveEndpointIdForeignKeyInIntegrityResolverTablesIntegrationTest.java

```diff
@@ -0,0 +1,171 @@
+package com.dotmarketing.startup.runonce;
+
+import com.dotcms.enterprise.license.bouncycastle.util.BigIntegers;
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.common.db.DotConnect;
+import com.dotmarketing.exception.DotDataException;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import static com.dotcms.util.CollectionsUtils.list;
+import static org.junit.Assert.assertFalse;
+
+
+public class Task05390RemoveEndpointIdForeignKeyInIntegrityResolverTablesIntegrationTest {
+
+    @BeforeClass
+    public static void prepare() throws Exception {
+        // Setting web app environment
+        IntegrationTestInitService.getInstance().init();
+    }
+
+    /**
+     * Method to Test: {@link Task05390RemoveEndpointIdForeignKeyInIntegrityResolverTables#executeUpgrade()}
+     * When: Run the Upgrade Task
+     * Should: Should remove the foreign key to all the Integrity Resolver tables and rename the column endpoint_id
+     * The Integrity Resolver tables are:
+     * - cms_roles_ir
+     * - folders_ir
+     * - structures_ir
+     * - htmlpages_ir
+     * - fileassets_ir
+     *
+     * After remove the constraint you should be allow to insert register in this table without has any register in publishing_end_point
+     */
+    @Test
+    public void constraintShouldNotExists() throws DotDataException {
```

Comment:
Codacy found an issue: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6190239)